hadoop-ozone/dist/src/main/compose/ozonesecure-ha/test-root-ca-rotation.sh

```diff
@@ -67,6 +68,7 @@ wait_for_execute_command scm4.org 240 "ozone admin cert info 4"
 
 #transfer leader to scm4.org
 execute_robot_test scm4.org -v "TARGET_SCM:scm4.org" scmha/scm-leader-transfer.robot
+execute_robot_test scm4.org -v PREFIX:"rootca" certrotation/root-ca-rotation-client-checks.robot
```

Comment:
rootca -> rootca2

I checked the failed robot test.  After leader is transferred to scm4.org,  leader OM will fail to allocate block for key upload since the three OM doesn't know scm4.org. We can move this root-ca-rotation-client-checks.robot test ahead of leader transfer.

Reply:
updates, thank you for checking into this.